nifi-nar-bundles/nifi-standard-bundle/nifi-standard-nar/src/main/resources/META-INF/NOTICE

```diff
@@ -248,6 +248,10 @@ The following binary components are provided under the Apache Software License v
       Apache MINA Core 2.0.16
       Copyright 2004-2016 Apache MINA Project
 
+  (ASLv2) Json-schema-validator 
+   The following NOTICE information applies:
+      Copyright (c) 2019 Network New Technologies Inc.
+
```

Comment:
This addition to the NOTICE file should not be necessary since JSON Schema Validator follows the standard Apache License 2.0 and does not include any specific notices except for third party libraries, which are already covered.